test/regression/cases/uds_dogstatsd_to_api_nomultivalue/experiment.yaml

```diff
@@ -0,0 +1,14 @@
+optimization_goal: ingress_throughput
+erratic: false
+
+environment:
+  DD_TELEMETRY_ENABLED: true
+
+profiling_environment:
+  DD_INTERNAL_PROFILING_ENABLED: true
+  DD_INTERNAL_PROFILING_UNIX_SOCKET: /var/run/datadog/apm.socket
+  DD_INTERNAL_PROFILING_DELTA_PROFILES: true
+  DD_INTERNAL_PROFILING_ENABLE_GOROUTINE_STACKTRACES: true
+  DD_INTERNAL_PROFILING_EXTRA_TAGS: experiment:uds_dogstatsd_to_api
```

Comment:
Copy paste error - this needs `_nomultivalue`

Reply:
Addressed in [d74f558](https://github.com/DataDog/datadog-agent/pull/19324/commits/d74f55823c583a014edf63f66040ec0ab9cbb118)